follower/consensus_follower.go

```diff
@@ -0,0 +1,150 @@
+package consensus_follower
+
+import (
+	"context"
+	"sync"
+
+	"github.com/onflow/flow-go/cmd"
+	access "github.com/onflow/flow-go/cmd/access"
+	"github.com/onflow/flow-go/consensus/hotstuff/notifications/pubsub"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// ConsensusFollower is a standalone module run by third parties which provides
+// a mechanism for observing the block chain. It maintains a set of subscribers
```

Comment:
so correct me if I am wrong here. With this setup we can do the following:

                       | Standalone (using `main.go`)   | Library   |
 -------------+-------------------------------+---------|
  Staked AN    |               ✅                                    |    ❌       |
--------------+------------------------------- |-------- |
  Unstaked AN|              ✅                                     |    ✅       |
--------------+--------------------------------+--------+

Reply:
That's correct :)